src/crypto.js

```diff
@@ -0,0 +1,183 @@
+const crypto = require('node:crypto')
+const UUID = require('uuid-1345')
+
+// authlib-3.4.40
+const YGGDRASIL_SESSION_PUBKEY_20220726 = crypto.createPublicKey(`\
```

Comment:
Is this a hardcoded file inside the client, or is this fetchable from the session server?

If its fetchable, can we add a boolean to fetchLatestSessionKey while still relying on the hardcoded one?

Reply:
it is a hardcoded file inside the authlib, which is a library that can change over time?, i just took it from there and converted it into a usable format